src/Model/AttendeeAvailability.php

```diff
@@ -28,6 +28,7 @@ class AttendeeAvailability extends Entity
 
     /**
     * Gets the attendee
+    * The type of attendee - whether it's a person or a resource, and whether required or optional if it's a person.
```

Comment:
In the future, can we separate pull requests by generated code and hand written to make it easier to review?